Algorithms/src/main/java/dynamic_programing/Factorial.java

```diff
@@ -0,0 +1,10 @@
+public class FactorialDP {
```

Comment:
Please take a look at the build again, it's due to the name of the file being different than the public class name